app/utils/file/file_picker/index.ts

```diff
@@ -109,7 +109,7 @@ export default class FilePickerUtil {
     };
 
     private getPermissionDeniedMessage = (source?: PermissionSource) => {
-        const sources = ['camera', 'storage', 'photo'];
+        const sources = ['camera', 'storage'];
         const deniedSource: PermissionSource = Platform.select({android: 'denied_android', ios: 'denied_ios'})!;
```

Comment:
Can you verify `denied_android` and `denied_ios` are only used here? If so, I think one of the issues is also about naming, and would be great if you could update that.

What I mean is that `denied_ios` and `denied_android` should be named `photo_android` and `photo_ios` instead, right? That way, this code would make more sense.